src/dmd/backend/cod3.d

```diff
@@ -230,39 +234,42 @@ void REGSAVE_save(ref REGSAVE regsave, ref CodeBuilder cdb, reg_t reg, uint *pid
              * reason. Need to fix.
              */
             op = STOUPD;
-        cdb.genc1(op,modregxrm(2, reg - XMM0, BPRM),FLregsave,cast(targ_uns) i);
+        cdb.genc1(op,modregxrm(2, reg - XMM0, BPRM),FLregsave,cast(targ_uns) idx);
     }
     else
     {
         if (!regsave.alignment)
             regsave.alignment = REGSIZE;
-        i = regsave.idx;
+        idx = regsave.idx;
         regsave.idx += REGSIZE;
         // MOV idx[RBP],reg
-        cdb.genc1(0x89,modregxrm(2, reg, BPRM),FLregsave,cast(targ_uns) i);
+        cdb.genc1(0x89,modregxrm(2, reg, BPRM),FLregsave,cast(targ_uns) idx);
         if (I64)
             code_orrex(cdb.last(), REX_W);
     }
     reflocal = true;
     if (regsave.idx > regsave.top)
         regsave.top = regsave.idx;              // keep high water mark
-    *pidx = i;
 }
 
-void REGSAVE_restore(ref REGSAVE regsave, ref CodeBuilder cdb, reg_t reg, uint index)
+/*******************************
+ * Restore `reg` from `regsave` area.
+ */
```

Comment:
Would be nice to flesh out this documentation with a params section as well.

Reply:
It says that it is complement to `REGSAVE_save` so it can be deduced that is has the same parameters